journeys/et/updateIP.ts

```diff
@@ -0,0 +1,25 @@
+import {Journey} from "types/journey"
+import {readFileSync, writeFileSync} from "fs";
+import {sep} from "path";
+import {execCommand} from "app/helpers";
+
+async function updateIP() {
+  const ip = (await execCommand("hostname -I | awk '{print $1}'")).stdout.trim();
+  updateRegion(process.env.ENGWALES_DEF_DIR, ip)
+  updateRegion(process.env.SCOTLAND_DEF_DIR, ip)
+}
+
+function updateRegion(regionDir: string, ip: string) {
+  const packageJsonPath = `${regionDir}${sep}package.json`;
+
+  const json = JSON.parse(readFileSync(packageJsonPath).toString())
+  // noinspection HttpUrlsUsage
```

Comment:
Guessing this is an intelliJ specific thing? vscode doesnt seem to complain without it

Reply:
Yep

Reply:
Is there a way to disable it that doesnt involve adding comments in code? Would like to avoid that where possible

Reply:
Yep, will do, probably nicer.